TodoLogic/src/components/TodoDescription.tsx

```diff
@@ -1,24 +1,35 @@
-import { useState } from "react";
+import { useContext } from "react";
+import { TodoContext } from "./Todo";
 
 interface TodoInf {
-  description: string;
+  id: number;
+  data: string;
+  check: boolean;
 }
-const TodoDescription = (props: TodoInf) => {
-  const [listItem, setListItem] = useState(false);
-  const Status = () => {
-    setListItem(!listItem);
+const todoDescription = (prop: TodoInf) => {
```

Comment:
React functional components should start with uppercase letter.
Rest JS functions should start with lowercase letter.